src/main/java/com/github/romankh3/image/comparison/ImageComparison.java

```diff
@@ -185,6 +190,39 @@ public ImageComparison(BufferedImage expected, BufferedImage actual) {
     }
 
     /**
+     * [work for issue #153]
+     * Draw rectangles which cover the regions of the difference pixels.
+     *
+     * @return the result of the drawing.
+     */
+    public ImageComparisonResult compareImages_BMP() {
```

Comment:
Please, rename method to be more java friendly. 
For example compareImagesBMP or similar.